### ISSUE TITLE
startup_usb_camera.sh

### ISSUE THREAD

**Issue by SodaBrad** [trevos3d/K1-USB-Cam#10]
Hi, I followed the scripts to install the USB camera functions, but only my USB camera is working and not my embedded K1 max camera. I have a feeling its due to the startup_USB camera.sh script being only for the single USB camera install and not dual.

The current script for startup_USB camera.sh is:
mjpg_streamer -b -i "/usr/lib/mjpg-streamer/input_uvc.so -d /dev/video4" -o "/usr/lib/mjpg-streamer/output_http.so -p 8080"

but looking at Kirbo's original script, I'm wondering whether it should be:

mjpg_streamer -b -i "/usr/lib/mjpg-streamer/input_uvc.so -d /dev/video5" -i "/usr/lib/mjpg-streamer/input_uvc.so -d /dev/video4" -o "/usr/lib/mjpg-streamer/output_http.so -p 8080"

from Kirbos startup_dual_cameras.sh where it includes video5 as well.

I am not too familiar with all this stuff, but I hope you could share your thoughts on this.

**Comment by rbferreira** [comment]
Good catch. The startup_usb_camera.sh is indeed identical to single_cam.sh and should be the dual_cam version when dual mode is installed. This is addressed by #19 which updates all three scripts consistently with the startup delay and safe kill fixes. The install script correctly copies the right template (single or dual) to startup_usb_camera.sh during installation.
